dapp/src/hooks/orangeKit/useBitcoinProvider.ts

```diff
@@ -8,7 +8,8 @@ export function useBitcoinProvider(): UseBitcoinProviderReturn {
   const connector = useConnector()
 
   return useMemo(() => {
-    if (!connector) return undefined
+    if (!connector || typeof connector.getBitcoinProvider !== "function")
+      return undefined
```

Comment:
I'm wondering why we added this condition here. This issue is also addressed [here](https://github.com/thesis/acre/pull/771).  Hook `useConnector` returns a connector from OrangeKit which should have an implemented method `getBitcoinProvider`.  Therefore, it seems to me that this is not right place for the fix.

Reply:
Wagmi sometimes returns connector w/o functions I'm not sure why - I noticed this in logs that we got an uncaught error and fixed it.